frog/imports/ui/GraphEditor/store/uiStore.js

```diff
@@ -1,14 +1,23 @@
 // @flow
-import { action, observable, computed } from 'mobx';
+import { action, observable, computed, autorun } from 'mobx';
 import { between, timeToPx, pxToTime } from '../utils';
 import { store } from './index';
 import type { Elem } from './store';
 import Operator from './operator';
 
 export default class uiStore {
+  constructor() {
+    autorun(() => {
+      if (this.selected) {
+        this.setStickySelected(this.selected);
+      }
+    });
```

Comment:
- Why do we need stickyselected ? 
- What does this piece of code in the constructor do? 

The sidePanel crashes if the graph is deleted while something is selected